sources/execDynawoAlgorithms.sh

```diff
@@ -77,12 +77,43 @@ setEnv() {
   export LD_LIBRARY_PATH=$DYNAWO_ALGORITHMS_INSTALL_DIR/lib:$LD_LIBRARY_PATH
 }
 
+find_and_call_timeline() {
+  if [ ! -d "$1" ]; then
+    return 1
+  fi
+  find $1 -name $2 | while read line; do
```

Comment:
line variable is a file ?

Reply:
not sure I get the question, this line looks like this for example: find /home/jobsfolder/ -name timeline.log

Reply:
oh ok I got it, yes line is a file do you want me to update the name?

Reply:
line is a line in the result lines of command find, so yes, line is a filename !

Reply:
Yes my comment was not clear but if the variable line is indeed a file better name filename I think

Reply:
fixed